src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -293,13 +292,13 @@ static void GetPanelSizes(
     private string GetSizeStorageKey()
     {
         var viewKey = ViewKey ?? NavigationManager.ToBaseRelativePath(NavigationManager.Uri);
-        return $"SplitterSize_{Orientation}_{viewKey}";
+        return $"Aspire_SplitterSize_{Orientation}_{viewKey}";
```

Comment:
I changed the keys to prevent the dashboard from using existing encrypted values. This means existing state is lost when updating to the next Aspire version, but it appears to be lost today anyway when protection keys change.

Going forward we can rely on these values being available long term.

Reply:
While we're at it, can we give this a better name? There might be multiple splitters in the dashboard. Let's make this more specific.

Reply:
viewKey is there to support multiple splitters. It is what lets resources/structured logs/trace detail to have splitters with different settings.

Reply:
Ok, makes sense.